taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -51,14 +51,9 @@ const AccumulateAllComponents = ({ id, taxonomyName, branchName }) => {
     let duplicateNode = null;
     if (node) {
       duplicateNode = { ...node[0] };
-      // Adding UUIDs for tags and properties
+      // Adding UUIDs for properties
       Object.keys(node[0]).forEach((key) => {
-        if (key.startsWith("tags") && !key.includes("ids")) {
-          duplicateNode[key + "_uuid"] = [];
-          duplicateNode[key].forEach(() => {
-            duplicateNode[key + "_uuid"].push(Math.random().toString());
-          });
-        } else if (key.startsWith("prop")) {
+        if (key.startsWith("prop")) {
           duplicateNode[key + "_uuid"] = [Math.random().toString()];
```

Comment:
TBH since this code is related to properties and this PR is related to tags, I did not try to understand it and let it for another PR; I think this PR is already big enough 😅